cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -821,7 +889,11 @@ fn get_fltreg_for_retval_systemv(
                 None
             }
         }
-        CallConv::WindowsFastcall | CallConv::BaldrdashWindows | CallConv::Probestack => todo!(),
+        CallConv::WindowsFastcall => match fltreg_idx {
+            0 => Some(regs::xmm0()),
+            _ => None,
```

Comment:
BTW, what is happening with multi-return? Is some code somewhere else panicking to avoid this case or should we be panicking here?

Reply:
With more return values, we use the return-area pointer, similar to what happens under SysV. This is, properly speaking, an extension (for both ABIs); but it's necessary for e.g. Wasm multi-value support.